web/guestbook-go/README.md

```diff
@@ -24,22 +24,22 @@ This example assumes that you have a working cluster. See the [Getting Started G
 
 ### Step One: Create the Redis master pod<a id="step-one"></a>
 
-Use the `examples/guestbook-go/redis-master-controller.yaml` file to create a [replication controller](https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/) and Redis master [pod](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/). The pod runs a Redis key-value server in a container. Using a replication controller is the preferred way to launch long-running pods, even for 1 replica, so that the pod benefits from the self-healing mechanism in Kubernetes (keeps the pods alive).
+Use the `examples/guestbook-go/redis-master-controller.yaml` file to create a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) and Redis master [pod](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/). The pod runs a Redis key-value server in a container. Using a deployment is the preferred way to launch long-running pods, even for 1 replica, so that the pod benefits from the self-healing mechanism in Kubernetes (keeps the pods alive).
 
-1. Use the [redis-master-controller.yaml](redis-master-controller.yaml) file to create the Redis master deployment in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
+1. Use the [redis-master-controller.yaml](redis-master-controller.yaml) file to create the Redis master deployment in your Kubernetes cluster by running the `kubectl apply -f` *`filename`* command:
 
     ```console
-    $ kubectl create -f guestbook-go/redis-master-controller.yaml
+    $ kubectl apply -f guestbook-go/redis-master-controller.yaml
```

Comment:
Let's try to unify the approach here, currently you're using 3 different paths when pointing to this file:
1. First instance uses full path `examples/guestbook-go/redis-master-controller.yaml`
2. Second one (above) uses just `redis-master-controller.yaml`
3. Here, you're using `guestbook-go/redis-master-controller.yaml`

I'm inclined to say to stick with this file context, so just use `redis-master-controller.yaml`

Let's use one consistently across all, so there is no confusion to the reader.